public/global.css

```diff
@@ -32,6 +32,8 @@ label {
 input, button, select, textarea {
 	font-family: inherit;
 	font-size: inherit;
+	/* Fix Padding In Chrome */
```

Comment:
Not sure the comment is necessary?

Reply:
+1 to this. None of the other styles have comments.

Reply:
removed. ty for the feedback